### ISSUE TITLE
No entity

### ISSUE THREAD

**Issue by microfix** [thomasloven/hass-browser_mod#178]
The mod do not generte new entity for me.
I have restartet the server multiple times.
No entity to find.
it's working, can use the popup function and All els. But No entity

**Comment by KTibow** [comment]
See anything in logs? Have you tried naming it? Are you sure you didn’t exclude it via config?

**Comment by microfix** [comment]
Have not use it before. The config is only
Browser_mod:
And every thing is working via lovelace. Use it daily. Installer it over a month ago. But I need the ability to interact with browser now. But No thing to be found. Have followed the installation guide. And install via hacs

**Comment by KTibow** [comment]
I think your translator isn't working. Could you explain everything you said?

**Comment by microfix** [comment]
Sure let me try again.

I only put in
Browser_mod:
In the configuration.yaml

I have used the mod for popups in lovelace whitout a problem, now i want to use my browser entity, but can't find it.

I installer the mod a while back, but have not been using the entity part of the mod before. Now i want to use my browser as a entity but they are not there.

I use about 10 devices, I must have missed something. If non of them is there.

**Comment by microfix** [comment]
I see a lot of this error in the log.

[]()
Traceback (most recent call last):

File "/usr/src/homeassistant/homeassistant/components/websocket_api/connection.py", line 95, in async_handle

handler(self.hass, self, schema(msg))

File "/config/custom_components/browser_mod/connection.py", line 42, in handle_update

devices[deviceID].update(msg.get("data", None))

File "/config/custom_components/browser_mod/connection.py", line 83, in update

self.sensor = self.sensor or create_entity(

File "/config/custom_components/browser_mod/helpers.py", line 47, in create_entity

adder = hass.data[DOMAIN][DATA_ADDERS][platform]

KeyError: 'sensor'

[]()

**Comment by microfix** [comment]
Got it to work.
Changed my configuration.yaml

Now it days:

Browser_mod:
  Prefix: "browser_"

**Comment by KTibow** [comment]
Could you close then please?
